src/BlogServiceProvider.php

```diff
@@ -46,9 +46,6 @@ public function bootingPackage()
         if (app()->runningInConsole() && ! app()->runningUnitTests()) {
             // Really just so route will appear in artisan route:list
             app()->booted(ServeBlog::blogRoutes());
-        } else {
-            // Deferred route registration if this is a Nova request
-            Nova::booted(ServeBlog::blogRoutes());
```

Comment:
If a Nova route is detected, there's no point in registering the blog route at all!